zerver/webhooks/github_webhook/view.py

```diff
@@ -401,6 +403,9 @@ def get_event(request, payload):
             return 'assigned_or_unassigned_pull_request'
         if action == 'closed':
             return 'closed_pull_request'
+        if action in ('labeled', 'unlabeled', 'review request', 'review request removed'):
```

Comment:
Maybe we should still capture fixtures for these and commit them, and have the tests verify that they do nothing?